project.py

```diff
@@ -136,24 +342,64 @@ def find_and_draw_edges(self, image, origin):
 
         img = origin.copy()
         cv.drawContours(img, [our_cnt], -1, (0, 255, 0), 3)
-        cv.imshow("img", img)
+        if(self.frame_counter % self.save_interval == 0):
```

Comment:
@Foxiy what's the point of doing the modulo operation twice? Line 348 will not allow `self.frame_counter` to become equal to `self.save_interval` and so this line can be rewritten as:

```python
if self.frame_counter == 0:
```

It also might be that I am missing something (but I am sure it's not the parentheses).